torch/testing/_internal/common_optimizers.py

```diff
@@ -542,6 +542,17 @@ def optim_inputs_func_adam(device, dtype=None):
             },
             desc="Tensor lr, Tensor betas, with capturable",
         ),
+        OptimizerInput(
+            params=None,
+            kwargs={
+                "lr": torch.tensor(0.001),
+                "betas": (torch.tensor(0.9), torch.tensor(0.99)),
```

Comment:
this test doesn't really test this PR. this should be
```suggestion
                "betas": (torch.tensor([[[0.9]]]), torch.tensor([[0.99]])),
```